packages/dev-server/index.js

```diff
@@ -47,7 +47,11 @@ function postConfig (context, util) {
         hot: true,
         hotOnly: true,
         historyApiFallback: true,
-        inline: true
+        inline: true,
+        // Disable verbose loggin in browser’s console
+        clientLogLevel: 'none',
```

Comment:
I’ve experimented a bit. This paragraph [is misleading](https://github.com/webpack/webpack-dev-server/issues/1038). Changing it to `error`.